firmware/src/src/targets/nrf52/btpararmt.cpp

```diff
@@ -340,7 +340,7 @@ static bool eir_found(struct bt_data *data, void *user_data)
         int err = bt_conn_le_create(addr, &btconparm, rmtconparms, &pararmtconn);
         if (err) {
           char buf[10];
-          LOGE("HT: Create conn failed (Error %s)", bytesToHex((uint8_t *)&err, 1, buf));
+          LOGE("Create conn failed (Error %s)", bytesToHex((uint8_t *)&err, 1, buf));
```

Comment:
earlier code had it so that 'HT:' needed to be in string for it to show in the gui serial console -- so, is this ok to remove here?

Reply:
I just believe you just missed this one, it's adding HT: twice. It's still being added in ht_serial_logger too.